main.js

```diff
@@ -326,14 +328,23 @@ function initMap() {
         if (!query.quiet) {
           var geocoder = L.Mapzen.geocoder(DEMO_API_KEY);
           geocoder.addTo(map);
-
+          
           // Duplicates existing bug behavior.
           // TODO: more appropriate links & messages.
+          
+          //get scene description if data defined otherwise intiialize to empty string
```

Comment:
Comments should be formatted with a space after `//`, also `intiialize` is spelled wrong, might as well fix it here